gwtp-core/gwtp-crawler-guice/src/main/java/com/gwtplatform/crawler/server/guice/ServiceKey.java

```diff
@@ -14,12 +14,13 @@
  * the License.
  */
 
-package com.gwtplatform.crawler.server;
+package com.gwtplatform.crawler.server.guice;
```

Comment:
Does it really matter if its a breaking change in this case? Since its a simple import change?